catalog/Data_Analysis/structures.yml

```diff
@@ -1,7 +1,5 @@
 name: Structures
 description: Projects for working with data structures such as vectors and data frames.
 projects:
-  - daru
```

Comment:
This is the analysis component of the DARu framework and [relocated in Analysis](https://github.com/rubytoolbox/catalog/pull/527/files#diff-b4b9d62617d722422b3e2081ea101cf0991f5fe2b7a1922aacf306e0fb5a6462R4).